faqs/galaxy/histories_shared.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Finding and working with "Histories shared with me"
+description: How to find and work on histories shared with you
+area: datasets
+layout: faq
+box_type: tip
+contributors: [jennaj, garimavs]
+---
+
+**To find histories shared with me:**
+- Log into your account.
+- Select **User**, in the drop-down menu, select **Histories shared with me**.
```

Comment:
```suggestion
1. Log into your account.
2. Select **User**, in the drop-down menu, select **Histories shared with me**.
```

Reply:
@garimavs does this suggestion make sense to you? I think this is the only pending change before merging

Reply:
Sorry for the delay @beatrizserrano, this suggestion sounds good to me. Will update it shortly!